### PR TITLE
[inner-2101] integration sql_log\show trace\show @@sql.xx\show @@cost_time

### PR DESCRIPTION
**Pull request by wenyh1**
Reason:  
  BUG #. or Improve #.  
Type:  
  BUG/Improve  
Influences：  
   fix xx

### CODE REVIEW

File: src/main/java/com/actiontech/dble/config/model/SystemConfig.java [hunk 1]
```diff
@@ -215,6 +215,7 @@ private SystemConfig() {
     private String routePenetrationRules = "";
 
     private int enableStatistic = 0;
+    private int enableStatisticAnalysis = 0;
```
Comment (ylinzhu) [inline src/main/java/com/actiontech/dble/config/model/SystemConfig.java:218]: toString needs to add this parameter

Comment (wenyh1) [reply src/main/java/com/actiontech/dble/config/model/SystemConfig.java:218]: get

File: src/main/java/com/actiontech/dble/statistic/trace/TrackProbe.java [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+    }
+
+    private void sqlTracking(Consumer<TraceResult> consumer) {
+        try {
+            if (isTrace) {
+                Optional.ofNullable(traceResult).ifPresent(consumer);
+            }
+        } catch (Exception e) {
+            // Should not affect the main task
+            LOGGER.warn("sqlTracking occurred: {}", e);
```
Comment (LUAgam) [inline src/main/java/com/actiontech/dble/statistic/trace/TrackProbe.java:193]: no need to add '{}'

Comment (wenyh1) [reply src/main/java/com/actiontech/dble/statistic/trace/TrackProbe.java:193]: get

File: src/main/java/com/actiontech/dble/server/response/ShowTables.java [hunk 3]
```diff
@@ -110,7 +110,7 @@ private static void parserAndExecuteShowTables(ShardingService shardingService,
         }
         RouterUtil.routeToSingleNode(rrs, node, null);
         ShowTablesHandler showTablesHandler = new ShowTablesHandler(rrs, shardingService.getSession2(), info);
-        shardingService.getSession2().setPreExecuteEnd(TraceResult.SqlTraceType.SINGLE_NODE_QUERY);
+        shardingService.getSession2().trace(t -> t.setPreExecuteEnd(TraceResult.SqlTraceType.MULTI_NODE_QUERY));
```
Comment (ylinzhu) [inline src/main/java/com/actiontech/dble/server/response/ShowTables.java:113]: use SINGLE_NODE_QUERY

File: src/main/java/com/actiontech/dble/services/manager/response/ShowHelp.java [hunk 4]
```diff
@@ -239,7 +237,9 @@ private static RowDataPacket getRow(String stmt, String desc, String charset) {
 
         HELPS.put("show @@statistic", "Turn off statistic information");
         HELPS.put("enable @@statistic", "Turn on statistic sql");
-        HELPS.put("disable @@statistic", "Turn off statistic sql");
+        HELPS.put("enable @@enableStatisticAnalysis", "Turn on statistic analysis sql('show @@sql.sum.user/table' or 'show @@sql.condition')");
+        HELPS.put("disable @@statistic", "Turn off statistic analysis sql('show @@sql.sum.user/table' or 'show @@sql.condition')");
```
Comment (ylinzhu) [inline src/main/java/com/actiontech/dble/services/manager/response/ShowHelp.java:241]: this is disable @@enableStatisticAnalysis

File: src/main/java/com/actiontech/dble/server/response/ShowTables.java (hunk 3 above)
Comment (wenyh1) [reply src/main/java/com/actiontech/dble/server/response/ShowTables.java:113]: get

File: src/main/java/com/actiontech/dble/services/manager/response/ShowHelp.java (hunk 4 above)
Comment (wenyh1) [reply src/main/java/com/actiontech/dble/services/manager/response/ShowHelp.java:241]: get

**Review by ylinzhu — DISMISSED** [review]
ok

**Comment by wenyh1** [comment]
Let's not rush the merger.
Wait for the requirements review before deciding.

**Comment by wenyh1** [comment]
alreafy have new pr

**Pull request closed without merging**